dir_class.py

```diff
@@ -159,3 +160,15 @@ def reading_data_from_text_file(self, file_name):
                 elif action == "u":
                     os.remove(file_path)
 
+    def new_file_name(self, file_name):
+        file_path = self.base_dir + os.sep + file_name
+        audio = EasyID3(file_path)
+        file_title = audio["title"]
+        if audio["title"] == "":
+            file_title = "-"
+        file_tracknumber = audio["tracknumber"]
+        if audio["tracknumber"] == "":
+            file_tracknumber = "-"
+        audio.save()
+        new_file_name = file_tracknumber + file_title
```

Comment:
Powinno być jeszcze rozszerzenie pliku - `mp3`.
Właściwie to najwygodniej będzie Ci skorzystać z f-stringów. Czyli coś w tym stylu:
`new_file_name = f"{file_tracknumber} - {file_title}.mp3`